ipn/local.go

```diff
@@ -776,7 +776,9 @@ func (b *LocalBackend) parseWgStatus(s *wgengine.Status) (ret EngineStatus) {
 		ret.WBytes += p.TxBytes
 	}
 	if len(peerStats) > 0 {
+		// [GRINDER STATS LINE] - please don't remove
```

Comment:
This is a little shouty and mysterious (for open source). Nobody will know what Grinder means.